src/tmpfiles/tmpfiles.c

```diff
@@ -843,6 +843,23 @@ static int fd_set_perms(Item *i, int fd, const struct stat *st) {
         return label_fix(path, 0);
 }
 
+static int path_open_parent_safe(const char *path) {
+        _cleanup_free_ char *dn = NULL;
+        int fd;
+
+        dn = dirname_malloc(path);
```

Comment:
if this is called "safe" this should really filter out bogus cases, i.e. specifying a path here such as "/" which doesn't really have a parent.